src/app/reporting/Engine.cpp

```diff
@@ -367,6 +355,20 @@ CHIP_ERROR Engine::CheckAccessDeniedEventPaths(TLV::TLVWriter & aWriter, bool &
                           current->mValue.mEndpointId, ChipLogValueMEI(current->mValue.mClusterId),
                           ChipLogValueMEI(current->mValue.mEventId), err == CHIP_ERROR_ACCESS_DENIED ? "ACL" : "ARL");
         }
+
+        Status status = CheckEventSupportStatus(path);
```

Comment:
Followup: CheckEventSupportStatus should not actually exist, as far as I can tell, and certainly should not be called from here.

Reply:
I was wrong; things are a bit complicated here.  Resolving, but we need to get the spec to define how subscription reports get generated and what checks are done at that point....